src/main/java/org/jvnet/hudson/test/DefaultConstructorChecker.java

```diff
@@ -19,9 +19,7 @@ public DefaultConstructorChecker(Class clazz) {
     protected void runTest() throws Throwable {
         try {
             clazz.getConstructor();
-        } catch (NoSuchMethodException e) {
-            throw new Error(clazz+" must have the default constructor",e);
-        } catch (SecurityException e) {
+        } catch (NoSuchMethodException | SecurityException e) {
```

Comment:
```suggestion
        } catch (NoSuchMethodException e) {
```

Not sure what that was there for, should never be thrown.